docs/use_cases/two-colors-serie.html

```diff
@@ -0,0 +1,245 @@
+</pre
+              >
+            </code>
+          </p>
+
+          <div id="htmlId">
+            <div class="border border-light position-relative">
+              <div class="chart_title">
+                <h4 class="display-4 mx-3 mb-1 mt-3">Title</h4>
+                <h5 class="display-5 mx-3 mb-1 mt-0">Sub-Title</h5>
+              </div>
+              <div id="barLine_holder">
+                <div
+                  id="barLine_chart"
+                  style="width: 100%; height: 50vh"
+                  class="position-relative"
+                ></div>
+              </div>
+              <div id="barLine_legend"></div>
+            </div>
+          </div>
+
+          <script>
+            addViewCode();
+          </script>
+        </div>
+      </div>
+      <script id="codeId">
+        ///////////////////////////////////////////////////
+        // Used data
+        ///////////////////////////////////////////////////
+
+        var goals = new Array(...new Array(12).keys()).map((i) => {
+          return 50 + Math.random() * 50;
+        });
+
+        var results = new Array(...new Array(12).keys()).map((i) => {
+          return 50 + Math.random() * 50;
+        });
+
+        var resultsOK = results.map((res, i) =>
+          res >= goals[i] ? results[i] : 0
+        );
+        var resultsNOK = results.map((res, i) =>
+          res < goals[i] ? results[i] : 0
+        );
+        var dates = new Array(...new Array(12).keys()).map((i) => {
+          var d = new Date();
+          d.setMonth(d.getMonth() - i);
+          return d.toLocaleDateString(undefined, {
+            month: 'short',
+            year: 'numeric',
+          });
+        });
+
+        // this is the data to be displayed
+        var dataOptions = {
+          xAxis: {
+            type: 'category',
+            data: dates,
+          },
+          yAxis: {},
+          series: [
+            {
+              data: resultsOK,
+              type: 'bar',
+              stack: 'result',
+            },
+            {
+              data: resultsNOK,
+              type: 'bar',
+              stack: 'result',
+            },
+            {
+              data: goals,
+              type: 'line',
+            },
+          ],
+          legend: {
+            data: ['Results OK', 'Results NOK', 'Goal'],
+          },
+        };
+
+        ///////////////////////////////////////////////////
+        // ODSCharts
+        ///////////////////////////////////////////////////
+        // Build the theme
+        var themeManager = ODSCharts.getThemeManager({
+          // Possibility to specify a Color out of ODS recommandations by given its value
+          // Cf https://ods-charts.netlify.app/api/interfaces/odschartsthemeoptions
+          // https://ods-charts.netlify.app/api/types/odschartscustomcategoricalcolor
+          // It can be the string value of the color
+          categoricalColors: ['#32C832', '#CD3C14', '#ff7900'],
```

Comment:
When you mentioned colours in https://github.com/Orange-OpenSource/ods-charts/pull/161#issuecomment-2009593856, you were referring to this line? That we should rather use the default colours from the library instead? If so, I definitely agree :)

Reply:
Dropped the custom colours in https://github.com/Orange-OpenSource/ods-charts/pull/161/commits/a8e8bdc29750e949b8ce1a3deb351169659bc3ee